app-network/ucx/autobuild/prepare

```diff
@@ -0,0 +1,2 @@
+abinfo "Running autogen.sh ..."
```

Comment:
This is done automatically in Autobuild (unless you specify `RECONF=0`).

Reply:
Again. I referred to OpenMPI's packaging scripts (app-scientific/openmpi) and I once thought this was the convention of AOSC.